network/p2p/libp2pNode_test.go

```diff
@@ -583,25 +582,28 @@ func (suite *LibP2PNodeTestSuite) CreateNodes(count int, handler network.StreamH
 	for i := 0; i < count; i++ {
 
 		name := fmt.Sprintf("node%d", i+1)
-		pkey, err := generateNetworkingKey(name)
+		key := generateNetworkingKey(suite.T())
 		require.NoError(suite.T(), err)
 
 		// create a node on localhost with a random port assigned by the OS
-		n, nodeID := suite.CreateNode(name, pkey, "0.0.0.0", "0", rootBlockID, handler, allowList)
+		n, nodeID := suite.CreateNode(name, key, "0.0.0.0", "0", rootBlockID, handler, allowList)
 		nodes = append(nodes, n)
 		nodeAddrs = append(nodeAddrs, nodeID)
 	}
 	return nodes, nodeAddrs
 }
 
-func (suite *LibP2PNodeTestSuite) CreateNode(name string, key crypto.PrivKey, ip string, port string, rootID string,
+func (suite *LibP2PNodeTestSuite) CreateNode(name string, key fcrypto.PrivateKey, ip string, port string, rootID string,
```

Comment:
With injectable libp2p node, we are literally passing dependencies at a higher level, where we are not necessarily dealing with libp2p keys. So, the new refactorings let creating a libp2p node (i.e., `NewLibP2PNode`) with flow keys and do the key translation to libp2p internally.